### PR TITLE
Update direction-column-reverse.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Replace incorrect image

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/styling/positioning-tips/direction-column-reverse.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/styling/positioning-tips/direction-column-reverse.md?ref=78cb1e848314fc5ae1568f1c25cfef34fe77bcb4)

File: web/styling/positioning-tips/direction-column-reverse.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ ___
 ```
 The four child boxes inside class parent will be displayed **vertically** on top of one another in reverse order, starting from the bottom of the parent element:
 
-![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/5cd4ebde7bebedb1168c64d4f3d8ee61.png)
+![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/9335ed930eba8f366028beb7d356daf4.png)
```
Comment (kapnobatai137) [inline web/styling/positioning-tips/direction-column-reverse.md:60]: could we edit this image a bit more? I think there's a lot of unnecessary purple background to the right @Stefan-Stojanovic

Comment (Stefan-Stojanovic) [inline web/styling/positioning-tips/direction-column-reverse.md:60]: Like this?
```suggestion
![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/c9bd35fa50bf1db13c2a2b28764e8e7a.png)
```

Comment (kapnobatai137) [reply web/styling/positioning-tips/direction-column-reverse.md:60]: yes, much better 💪

**Pull request merged**